pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[project]
+name = "vobject"
+dynamic = ["version"]
+description = "A full-featured Python package for parsing and creating iCalendar and vCard files"
+authors = [
+    { name = "Jeffrey Harris", email = "jeffrey@osafoundation.org" },
+]
+maintainers = [
+    { name = "David Arnold", email = "davida@pobox.com" },
+]
+license = { text = "Apache" }
+readme = "README.md"
```

Comment:
FWIW the current README is a little long to be the PyPI full description. Might consider a follow-up change to curate some of the details into separate files.

See also: I found https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/ has some useful pointers on that.

Reply:
The link you provide looks like it only talks about file formats and how to integrate your readme into setup.py (so just what `readme = "README.md"` does in the pyproject).

I also do not like the readme. It contains all the documentation that is available, which sadly is just some examples. Such things should be addressed in a follow up PR. This is only the setup.py -> pyproject conversion.